migrations/2022-11-21-204256_user-following/up.sql

```diff
@@ -0,0 +1,14 @@
+-- create user follower table with two references to persons
+create table person_follower (
+    id serial primary key,
+    person_id int not null,
+    follower_id int not null,
+    published timestamp not null default now(),
+    pending boolean not null default false,
+    unique (follower_id, person_id)
+);
+
+alter table person_follower add foreign key (follower_id) references person on update cascade on delete cascade;
+alter table person_follower add foreign key (person_id) references person on update cascade on delete cascade;
+
+alter table community_follower alter column pending set not null;
```

Comment:
hrm... I think this should be okay, since there is an existing default value for the column.

Reply:
There is just no reason why it should be nullable.